http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -322,6 +326,52 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
               )
               .run(authContext)
           } yield ()
+        case AppType.WorkflowsApp =>
+          for {
+            // Get the batch account key
+            batchAccount <- azureBatchService.getBatchAccount(params.landingZoneResources.batchAccountName,
+                                                              params.cloudContext
+            )
+
+            batchAccountKey = batchAccount.getKeys().primary
+
+            // Storage container is required for Workflows app
```

Comment:
I don't actually know if this is true. This part was taken from CoA chart creation.

Reply:
Did more digging and it looks like this is used more on the cromwell-runner side of things, so it may be safe to not include here.

Reply:
Hmm, I do see the workflows app using the storage account but this may be tied into the question of "what configs does a reader cromwell instance actually need to run properly". I don't see the workflows app using the batch account key.